site/src/components/DeploySettingsLayout/Badges.tsx

```diff
@@ -115,6 +134,11 @@ const useStyles = makeStyles((theme) => ({
     backgroundColor: theme.palette.error.dark,
   },
 
+  warnBadge: {
+    border: `1px solid ${theme.palette.warning.light}`,
+    backgroundColor: theme.palette.warning.dark,
+  },
```

Comment:
Nice! We do now support [writing styled components](https://emotion.sh/docs/styled) as opposed to classes, which we may want to lean into so we can get rid of `makeStyles` eventually. Not something that needs to be changed here - just a heads up.

Reply:
Huh TIL. Will have to see some examples in the future.